api/src/main/java/org/pmiops/workbench/access/AccessTierServiceImpl.java

```diff
@@ -205,6 +201,20 @@ private void removeFromAuthDomainIdempotent(DbUser dbUser, DbAccessTier accessTi
     }
   }
 
+  @Override
+  public List<DbUser> getAllRegisteredTierUsers() {
+    return userAccessTierDao.getAllByAccessTier(getRegisteredTierOrThrow()).stream()
+        .filter(uat -> uat.getTierAccessStatusEnum() == TierAccessStatus.ENABLED)
```

Comment:
nit: `Prefer equals() over ==`
Copied from a Google's java style guide: 
`Both choices above produce the correct result. However, we prefer code that is not just correct, but self-evidently correct. It's best if the reader can know the code is safe from bugs without having to look up whether the type of e1 and e2 is an enum. This makes equals the better choice (e1.equals(e2) when e1 is not nullable; Objects.equals(e1, e2) otherwise).`

Reply:
Disagree for enums.  `==` is required for typechecking.

This comparison to the wrong enum type correctly causes a compile error:
```
.filter(uat -> uat.getTierAccessStatusEnum() == WorkspaceActiveStatus.ACTIVE)
```

This does not, leaving us to catch the error through testing:
```
.filter(uat -> uat.getTierAccessStatusEnum().equals(WorkspaceActiveStatus.ACTIVE))
```

Reply:
Right, using == does have a small advantage in that it fails to compile if the types don't match. However, thanks to Intellij/spotbugs, we have the very protection when calling ether equals method.